client/src/event/Auth.vue

```diff
@@ -7,29 +7,33 @@
     <form v-else-if="event" @submit.prevent="onSubmit">
       <div class="row">
         <div class="col">
-          <label>This form is to register or edit a project for Coolest Projects International 2018 which will be held on Saturday 26th May 2018 in RDS Simmonscourt, Dublin, Ireland. Just one person should register for each project.</label>
+          <label v-if="isOpen">This form is to register or edit a project for Coolest Projects International 2018 which will be held on Saturday 26th May 2018 in RDS Simmonscourt, Dublin, Ireland. Just one person should register for each project.</label>
+          <label v-if="!isOpen && !isFrozen">You can now only edit already registered projects for {{ event.name }}. Enter your email and you will get a link to edit your project.</label>
+          <label v-else-if="isFrozen">The registration for {{ event.name }} has now closed. Please contact {{ event.email }} if you have any questions.</label>
         </div>
       </div>
-      <div class="row">
+      <div class="row" v-if="!isFrozen">
         <div class="col">
           <input type="email" v-model="email" class="full-width-block" placeholder="you@email.com" required />
```

Comment:
Without backend checks, this will bring the user to create a new project if they haven't already got one.